WeakAuras.lua

```diff
@@ -1810,6 +1810,28 @@ function WeakAuras.Add(data)
   -- end
 end
 
+
+function removeSpellNames(data)
```

Comment:
Ah, it would be nice if all the files would follow one indentation..

I'll wait until you are through with the review and then update all of the patches, okay?